package.json

```diff
@@ -20,7 +20,7 @@
   },
   "homepage": "https://github.com/ksachdeva/angular2-swing#readme",
   "dependencies": {
-    "swing": "^4.3.0"
+    "swing": "^3.1.0"
```

Comment:
Latest release is 4.3 - https://github.com/gajus/swing/releases

@gajus can you please confirm this?